lib/application-processing/resolvers.ts

```diff
@@ -874,15 +876,47 @@ export const updateApplicationProcessingGenerateInvoice: Resolver<
   }
 
   // Generate application invoice
-  generateApplicationInvoicePdf(
+  const pdfDoc = generateApplicationInvoicePdf(
     application,
     session,
     // TODO: Remove typecast when backend guard is implemented
     application.applicationProcessing.appNumber as number,
     invoice.invoiceNumber
   );
 
-  return { ok: true };
+  // Upload pdf to s3
+  const s3InvoiceKey = `invoices/${randomUUID()}/${invoice.invoiceNumber}.pdf`;
+  let uploadedPdf;
+  let signedUrl;
+  try {
+    uploadedPdf = await serverUploadToS3(pdfDoc, s3InvoiceKey);
+    signedUrl = getSignedUrlForS3(uploadedPdf.key);
+  } catch (error) {
+    throw new ApolloError(`Error uploading invoice pdf to AWS: ${error}`);
+  }
```

Comment:
set the ttl of the url to be max of 7 days by default. We can change this by passing in a duration param to this helper function

Reply:
Sounds good, could you please add a comment here for clarification (in case we need to change it later)?